contributing/communications.md

```diff
@@ -17,27 +17,38 @@ Hello, so you're interested in helping out with our communications? We're pretty
 
-### Write Like a Human...
 
-Be professional, yes, but also conversational. We don't want to sound like robots, and should certainly never be communicating on behalf of the Pa11y project in unfeeling business-speak. At all times our users should feel like they're talking to a regular human.
-
-### ...But a Nice Human
+### Tone of Voice Guidelines
 
+- Write like a human  
+Be professional, yes, but also conversational. We don't want to sound like robots, and should certainly never be communicating on behalf of the Pa11y project in unfeeling business-speak. At all times our users should feel like they're talking to a regular human.
+- But a Nice Human  
 Be friendly, even if the user you're speaking to is being difficult (excluding when there's a [Code of Conduct violation](#code-of-conduct)). Our users are giving up time and energy to make their sites more accessible, and potentially contributing back.
+- Be Thankful...  
+Make sure you thank anyone who's made a contribution or is even simply _using_ Pa11y (when it feels appropriate).
+- ...Especially to our Contributors  
+Our contributors are amazing – they're giving up their time to make the web as a whole more accessible. Our gratefulness should definitely come across, as long is it's not overbearing!
+- Even in Documentation  
+We're not saying that the documentation should read like a conversation, as this would make technical content imprecise. However introductions and any signposting should be friendly and helpful.
 
-### Be Thankful...
-
-Not in a cheesy Thanksgiving "I'm thankful for" kinda way; but make sure you thank anyone who's made a contribution or is even simply _using_ Pa11y (when it feels appropriate).
-
-### ...Especially to our Contributors
 
-Our contributors are amazing – they're giving up their time to make the web as a whole more accessible. Our gratefulness should definitely come across, as long is it's not overbearing!
+### Plain English guidelines
 
-### Even in Documentation
+Follow Plain English guidelines to help safeguard the accessibility of your written content. It'd be ironic for an a11y project to produce documents that are hard to understand! All relevant guidelines for [writing well][writing-well] apply. In particular:
 
-We're not saying that the documentation should read like a conversation, that would probably be really jarring and unhelpful when describing technical things. However introductions and any signposting should be friendly and helpful.
+- Be concise  
+  - Check any sentences with more than 25 words to see if you can split them to make them clearer. 
+  - Use contractions (e.g. "can't", "it's").
+  - Be brisk but not terse.
+- Be specific
+  - Use the active voice instead of the passive voice (e.g. "NPM will install the dependencies" instead of "the dependencies will be installed").
```

Comment:
Maybe we could link to [Hemingway](http://www.hemingwayapp.com/) for help on this?